modules/nlu/src/backend/engine.ts

```diff
@@ -158,10 +172,10 @@ export default class Engine implements NLUEngine {
       const intents = await ProcessIntents(
```

Comment:
the `if (!model.data.output) {` right above was not correctly renamed and still references to the previous property output which is now a step datastructure.

Reply:
check one line above my comment, I can't comment it because it was not changed...